yarn-project/aztec-cli/src/index.ts

```diff
@@ -146,8 +146,8 @@ export function getProgram(log: LogFn, debugLogger: DebugLogger): Command {
   program
     .command('deploy')
     .description('Deploys a compiled Noir contract to Aztec.')
-    .requiredOption(
-      '-c, --contract-abi <file>',
+    .argument(
+      '<abi>',
```

Comment:
I like this change, but heads up this impacts other guides, as well as the one in the up-dev page. We'll need to update those as well. Let's include them in this PR.

Reply:
This is maybe a good argument for writing all code snippets as part of testable code files (like you did earlier this week for some other docs, Santiago), so that the CI can then test that the code snippets are all up-to-date?

Reply:
Agree. I did fail to get them into the CI though (build-system beat me again), but can revisit it next week.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/1633/commits/9dea419a43ca4bc74ddd8fecb7bd462c7cb6eca6

I will send a PR to the sandbox website as well.